mimesis/builtins/pl.py

```diff
@@ -0,0 +1,91 @@
+"""Specific data provider for Poland (pl)."""
+
+from typing import Optional
+
+from mimesis.builtins.base import BaseSpecProvider
+from mimesis.enums import Gender
+from mimesis.providers import Datetime
+from mimesis.typing import DateTime
+
+__all__ = ['PolandSpecProvider']
+
+
+class PolandSpecProvider(BaseSpecProvider):
+    """Class that provides special data for Poland (pl)."""
+
+    class Meta:
+        """The name of the provider."""
+
+        name = 'poland_provider'
+
+    def nip(self) -> str:
+        """Generate random valid 10-digit NIP.
+
+        :return: valid 10-digit NIP
+        """
+        nip_digits = [int(d) for d in str(self.random.randint(101, 998))]
+        for _ in range(6):
+            nip_digits.append(self.random.randint(0, 9))
+        nip_coefficients = (6, 5, 7, 2, 3, 4, 5, 6, 7)
+        sum_v = sum(
+            map(lambda x: x[0] * x[1], zip(nip_coefficients, nip_digits)))
+        checksum_digit = sum_v % 11
+        if checksum_digit > 9:
+            checksum_digit = 0
+        nip_digits.append(checksum_digit)
+        return ''.join(str(d) for d in nip_digits)
+
+    def pesel(self,
+              birth_date: Optional[DateTime] = None,
+              gender: Optional[Gender] = None) -> str:
+        """Generate random 11-digit PESEL.
+
+        :param birth_date: initial birth date (optional)
+        :param gender: gender of person
+        :return: valid 11-digit PESEL
+        """
+        date_object = birth_date or Datetime().datetime(1800, 2299)
```

Comment:
```
date_object = birth_date
if not date_object:
    date_object = Datetime().datetime(1950, 2022)
```

Reply:
What's wrong with `date_object = birth_date or Datetime().datetime(1800, 2299)`? 
I like it, but I might be biased because I proposed it in the previous PR. ;)

Reply:
Maybe @lk-geimfari meant that if both parts will be true then date_object can be `birth_date` or can be 'Datetime().datetime(1950, 2022)` (implicit declaration sort of).

Reply:
No, that's not how `A or B` works in Python (or in most other languages). https://docs.python.org/3/reference/expressions.html#boolean-operations